opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/ChildDao.java

```diff
@@ -132,29 +131,27 @@ public static boolean isChild(String baseEntityID) {
         return res.get(0) > 0;
     }
 
-    public static String getBaseEntityID(String identifierType, String id) {
-        String selection = "select base_entity_id from ec_child where ? = ?";
-        String[] selectionArgs = new String[]{identifierType, id};
+    public static String getBaseEntityID(String thinkMdId) {
+        String selection = " thinkmd_id = ? ";
+        String[] selectionArgs = new String[]{thinkMdId};
         return ChildDao.queryColumnWithEntityId(selection, selectionArgs, BASE_ENTITY_ID);
     }
```

Comment:
shouldn't this reuse the older method `getBaseEntityID(String identifierType, String id)` and pass the params `getBaseEntityID("thinkmd_id", thinkMdId)`

Reply:
Done.